tests/mocha-no-side-effect-code/configured-factory-methods/test.ts.lint

```diff
@@ -0,0 +1,7 @@
+const x = RestDataFactory.createSocialProfile()
```

Comment:
These are supposed to be test cases for `ignors` option [link](https://github.com/Microsoft/tslint-microsoft-contrib/blob/9084d549f575db25c5818d6196ab8d381f9253fd/src/tests/MochaNoSideEffectCodeRuleTests.ts#L332)